src/main/telemetry/telemetry.h

```diff
@@ -59,5 +59,4 @@ bool telemetryDetermineEnabledState(portSharing_e portSharing);
 
 void telemetryUseConfig(telemetryConfig_t *telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM)
-
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_CRSF)
```

Comment:
> It could also be argued that for tightly integrated solutions like CRSF, where users wouldn't be expected to run CRSF telemetry with a serialRx that is not CRSF, it would be more userfriendly to go the way that JETIEXBUS goes and don't even show the telemetry protocol in the UI, but activate it automatically whenever the serialRx protocol is selected.

Agreed. It is unlikely that anyone would want to use CRSF telemetry unless they were using a CRSF RX, so not making it shareable makes setup easier. If there is user demand in future, we can change it. @blckmn are you agreed?